src/fragments/gen2/quickstart/deploy-and-host.mdx

```diff
@@ -8,7 +8,7 @@ If you already have your project remotely hosted in a Git repository, you can sk
 
 ### Connect branch in the Amplify console
 
-1. To get started with Gen 2, log in to the [AWS console](https://console.aws.amazon.com/console/home?#amplify) and navigate to your preferred AWS Region. (The Amplify console is available in [19 AWS Regions](https://docs.aws.amazon.com/general/latest/gr/amplify.html#amplify_region)).
+1. To get started with Gen 2, log in to the [Amplify console](https://console.aws.amazon.com/amplify/home?) and navigate to your preferred AWS Region. (The Amplify console is available in [19 AWS Regions](https://docs.aws.amazon.com/general/latest/gr/amplify.html#amplify_region)).
 2. From the **Public Preview** banner, choose **Try Amplify Gen 2**.
 
 ![Amplify Gen 2 console showing the "Public Preview" banner with "Try Amplify Gen 2" button](/images/gen2/getting-started/console1.png)
```

Comment:
For standardization, we should also consider updating the alt text from `Amplify Gen 2 console` to `Amplify console`.